python/repeated.c

```diff
@@ -326,35 +328,37 @@ static int PyUpb_RepeatedContainer_SetSubscript(
   }
   for (Py_ssize_t i = 0; i < count; i++, idx += step) {
     upb_msgval msgval;
-    PyObject* item = PySequence_GetItem(seq, i);
+    item = PySequence_GetItem(seq, i);
```

Comment:
now I think the success case is missing decref's on item for every iteration except the last

Reply:
Done.